docs/FAQ.md

```diff
@@ -34,9 +34,6 @@ Excellent question. Please visit the [High Contrast Support](HighContrastSupport
 #### Q. Where can I find out more about a specific rule?
 Explanations for all rules can be found at [Rules explained].
 
-#### Q. How do I create a new automated scan rule?
```

Comment:
Do we want to keep the question and point them to the new docs?

Reply:
@RobGallo - You did add a question to this same effect in this doc and I think that covered this question well so I thought of deleting it. I will go ahead and merge this and if you still think we should solve this a different way then I am more than happy to open a new PR after we discuss.